scripts/scriptassist2.pl

```diff
@@ -0,0 +1,1293 @@
+use strict;
+use vars qw($VERSION %IRSSI);
+use utf8;
+use POSIX;
+use File::Glob qw/:bsd_glob/;
+use CPAN::Meta::YAML;
+use File::Fetch;
+use Time::Piece;
+use Digest::file qw/digest_file_hex/;
+use Digest::MD5 qw/md5_hex/;
+use Text::Wrap;
+use JSON::PP;
+use Cwd;
+#use debug;
+
+use Irssi;
+
+$VERSION = '0.01';
+%IRSSI = (
+   authors     => 'bw1',
+   contact     => 'bw1@aol.at',
+   name        => 'scriptassist2',
+   description => 'keeps your scripts on the cutting edge',
+   license     => 'lgpl',
+   url         => 'https://scripts.irssi.org/',
+   changed     => '2021-02-13',
+   modules     => 'POSIX File::Glob CPAN::Meta::YAML File::Fetch Time::Piece Digest::file Digest::MD5 Text::Wrap'.
+                  'JSON::PP Cwd',
+   commands    => 'scriptassist2',
+   selfcheckcmd=> 'scriptassist2 selfcheck',
+);
+
+my $help = << "END";
+%9Name%9
+  $IRSSI{name}
+%9Version%9
+  $VERSION
+%9Description%9
+  $IRSSI{description}
+%9Commands%9
+  /$IRSSI{name} check
+      Check all loaded scripts for new available versions
+  /$IRSSI{name} update <script|all>
+      Update the selected or all script to the newest version
+  /$IRSSI{name} search <query>
+      Search the script database
+  /$IRSSI{name} info <scripts>
+      Display information about <scripts>
+  /$IRSSI{name} ratings <scripts|all>
+      Retrieve the average ratings of the the scripts
+  /$IRSSI{name} top <num>
+      Retrieve the first <num> top rated scripts
+  /$IRSSI{name} new <num>
+      Display the newest <num> scripts
+  /$IRSSI{name} rate <script>
+      Rate the script if you like it
+  /$IRSSI{name} contact <script>
+      Write an email to the author of the script
+      (Requires OpenURL)
+  /$IRSSI{name} cpan <module>
+      Visit CPAN to look for missing Perl modules
+      (Requires OpenURL)
+  /$IRSSI{name} install <script>
+      Retrieve and load the script
```

Comment:
It makes sense to comment here :)
I'd like a flag too, `/script install -auto trackbar` to make script autorun

Reply:
* install -auto :heavy_check_mark: